ray-operator/config/rbac/editor_role.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kuberay-edit-role
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+rules:
+- apiGroups:
+  - ray.io
+  resources:
+  - rayjobs
+  - rayjobs/status
+  - rayjobs/finalizers
+  - rayclusters
+  - rayclusters/status
```

Comment:
It's better to have the sub resources grouped in another item as some of the verbs like `deletecollection` do not apply.